apps/stacker/app.js

```diff
@@ -0,0 +1,140 @@
+const TICKRATE = 80;
+const BLOCK_SIZE = 16;
+const BLOCK_COLOR_PRIMARY = "#ff0000";
+const BLOCK_COLOR_SECONDARY = "#ffff00";
+const GAMEBOARD_X = 16;
+const GAMEBOARD_WIDTH = g.getWidth() - 16 - BLOCK_SIZE;
+const START_X = GAMEBOARD_X;
+const START_Y = g.getHeight() - BLOCK_SIZE - 1;
+const START_LENGTH = 5;
+var startTime = 0;
```

Comment:
May I suggest adding this code or similar to differentiate between Bangle.js 1 and 2 center buttons?
```
const HARDWARE_VERSION = process.env.HWVERSION;
const BUTTON = HARDWARE_VERSION === 2 ? BTN : BTN2;
```
Then replace all other instances of `BTN` with `BUTTON`